views-jstachio/src/test/java/io/micronaut/views/jstachio/PersonMessageBodyWriter.java

```diff
@@ -0,0 +1,48 @@
+package io.micronaut.views.jstachio;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.core.annotation.Order;
+import io.micronaut.core.io.Writable;
+import io.micronaut.core.order.Ordered;
+import io.micronaut.core.type.Argument;
+import io.micronaut.core.type.MutableHeaders;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.http.body.MessageBodyWriter;
+import io.micronaut.http.codec.CodecException;
+import jakarta.inject.Singleton;
+import java.io.IOException;
+import java.io.OutputStream;
+
+@Requires(property = "spec.name", value = "HtmlControllerSpec")
+@Order(Ordered.LOWEST_PRECEDENCE)
+@Produces(MediaType.TEXT_HTML)
+@Singleton
+public class PersonMessageBodyWriter implements MessageBodyWriter {
```

Comment:
add generics

Reply:
I did not add the type in this test intentionally. I did not want to hit: 

https://github.com/micronaut-projects/micronaut-core/blob/4.1.x/http/src/main/java/io/micronaut/http/body/DefaultMessageBodyHandlerRegistry.java#L144-L152

but: 

https://github.com/micronaut-projects/micronaut-core/blob/4.1.x/http/src/main/java/io/micronaut/http/body/DefaultMessageBodyHandlerRegistry.java#L157-L160

Reply:
i've changed it